### ISSUE TITLE
Screen Reader: Global: Bottom tab Account is announced with incorrect label

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77360]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack Android, open the app
2. On the Inbox screen, swipe right to focus the Account bottom bar and note how it is announced

This is a global issue occurring in all bottom navigation bar tabs

## Expected Result:
The tab should be announced with its visible name, e.g.:
- 'Account'

## Actual Result:
The tab 'Account' is announced as 'My settings'/

## Area issue was found in:
Inbox

## Failed WCAG checkpoints
WCAG 2.5.3 Label in Name
https://www.w3.org/WAI/WCAG22/Understanding/label-in-name

## User impact:
Speech-input users can directly activate controls on a page with fewer surprising changes of focus. Text-to-speech users will have a better experience because the labels that they hear match the visible text labels that they see on the screen.

## Suggested resolution:
Update the accessibilityLabel to match the visible text, e.g.:
- 'Account'
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment(s)
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/b2e275d7-0020-4240-9f87-dd2db3186d65)

https://github.com/user-attachments/assets/dbcf77a5-34a9-4262-8bcd-f28dfef4825f

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005819
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
dupe of 

- #75469
